packages/react/src/components/Table/TableToolbar/_table-toolbar.scss

```diff
@@ -93,13 +93,15 @@ html[dir='rtl'] {
   display: flex;
   align-items: center;
   text-align: left;
+  min-width: 0;
 
   [dir='rtl'] & {
     text-align: right;
   }
 
   svg {
     margin-right: $spacing-03;
+    min-width: $spacing-05;
```

Comment:
I think it is shrinking because it is in a flex container, so I would use `flex-shrink: 0;` instead of adding a min-width

Reply:
Ah! Of course. Thanks!